h/celery.py

```diff
@@ -42,6 +42,10 @@
             'task': 'h.tasks.cleanup.purge_expired_auth_tickets',
             'schedule': timedelta(hours=1)
         },
+        'purge-expired-authzcodes': {
+            'task': 'h.tasks.cleanup.purge_expired_authz_codes',
+            'schedule': timedelta(hours=1)
+        },
```

Comment:
Not so fast, I wanted to it after this is out. But since you insist: hypothesis/h-periodic#7 :stuck_out_tongue_winking_eye: